apps/examples/testcase/le_tc/filesystem/tc_fs_procfs.c

```diff
@@ -34,15 +34,16 @@
 #include <sys/stat.h>
 #include "tc_common.h"
 
-#define PROC_MOUNTPOINT "/proc"
+#define PROCFS_TEST_MOUNTPOINT "/test_proc"
```

Comment:
no, we gonna use /proc like a linux

Reply:
But it is good to do like this. So that there will not be any dependency on "/proc" mount point used by kernel